test/e2e/storage/persistent_volumes-local.go

```diff
@@ -669,11 +680,11 @@ func createVolumeConfigMap(config *localTestConfig) {
 	Expect(err).NotTo(HaveOccurred())
 }
 
-func createBootstrapperPod(config *localTestConfig) {
+func createBootstrapperJob(config *localTestConfig) {
```

Comment:
The job spec requires some more changes: https://github.com/kubernetes-incubator/external-storage/pull/313